### ISSUE TITLE
storage.ManagementPoliciesClient.CreateOrUpdate Failed with RequestIsNotValidJson

### ISSUE THREAD

**Issue by stuartleeks** [Azure/azure-sdk-for-go#5190]
### Bug Report

- import path of package in question, e.g. `.../services/storage/mgmt/2019-04-01/storage`
- SDK version `31.0.0`
  - Specify the exact commit if possible; one way to get this is the REVISION
    column output by `dep status "github.com/Azure/azure-sdk-for-go`.
- output of `go version`: `go version go1.12.6 linux/amd64`

Attempting to create a `ManagementPolicyRule` results in an error:

```
Error creating Azure Storage Management Policy : storage.ManagementPoliciesClient#CreateOrUpdate: Failure responding to request: StatusCode=400 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="RequestIsNotValidJson" Message="The request cannot be interpreted as valid Json: {}."
```

Repro code [here](https://gist.github.com/stuartleeks/19444b0e462062f1d4b4e14205ac5a66)

**Comment by stuartleeks** [comment]
It looks like this is an issue with the swagger for storage: https://github.com/Azure/azure-rest-api-specs/issues/6549

**Comment by stuartleeks** [comment]
Found [another issue ](https://github.com/Azure/azure-rest-api-specs/issues/6554) that also needs addressing for Management Policies

**Comment by stuartleeks** [comment]
Bump - the swagger has been updated (see [here](https://github.com/Azure/azure-rest-api-specs/issues/6554)) - what needs to happen to create a new release of the SDK from the updated swagger?

Thanks!

**Comment by ArcturusZhang** [comment]
Hi @stuartleeks , this issue should be resolved in version 32.0.0

**Comment by amishra-dev** [comment]
Kindly reactivate if you find the issue is not resolved in v32.0.0

**Comment by stuartleeks** [comment]
Apologies - I was juggling this in tandem with some other work. I appreciate your help getting this fixed and release - the Terraform provider work that this unblocked has now been released :-) Thanks!
